src/main/java/hockeyapp/HockeyappApplication.java

```diff
@@ -46,7 +48,8 @@
     @DataBoundConstructor
     public HockeyappApplication(String apiToken, String appId, boolean notifyTeam,
                                 String filePath, String dsymPath, String libsPath,
-                                String tags, boolean downloadAllowed, OldVersionHolder oldVersionHolder,
+                                String tags, String teams, boolean mandatory, 
+                                boolean downloadAllowed, OldVersionHolder oldVersionHolder,
```

Comment:
Hi, the params where reordered since it looked like they where ordered based on functionality. Thereby teams and tags beside each other, and mandatory and downloadAllowed beside each other.